Plots/Bar/NCL_bar_5.py

```diff
@@ -0,0 +1,235 @@
+    year_iend = int(np.where(np.round(date_frac, 3) == year_end)[0])
+
+    # Create each bar chart where it is red if it is above 0 and blue if below
+    ax_dict[ax].bar(date_frac[year_istart:year_iend],
+                    sst[year_istart:year_iend],
+                    align='edge',
+                    edgecolor='black',
+                    color=[
+                        'red' if (value > 0) else 'blue'
+                        for value in sst[year_istart:year_iend]
+                    ],
+                    width=.08,
+                    linewidth=1)
+    # Turn off axis so it is not visible
+    ax_dict[ax].axis("off")
+
+# Create right y axis by cloning the left side
+axRHS = ax0.twinx()
+
+# Make a variable for the degree symbol
+degree = u"\u00b0"
+
+# Use geocat.viz.util convenience function to add titles to the center and right of the plot axis
+gvutil.set_titles_and_labels(ax0,
+                             maintitle="Monthly SST Anomalies for Nino-3",
+                             maintitlefontsize=25,
+                             righttitle=("(" + degree + "C)"),
+                             righttitlefontsize=18)
+# Add center title
+ax0.text(0.38, 1.05, 'Warm Events', fontsize=18, transform=ax0.transAxes)
+
+# Use geocat.viz.util convenience function to add major tick lines
+gvutil.add_major_minor_ticks(ax0, x_minor_per_major=1, y_minor_per_major=1)
+
+# Create the tick labels for the left axis
+left_y_ticks = []
+i = 0
+left_y_ticks.append("")
+while i < num_subplots:
+    left_y_ticks.append("-1.0")
+    left_y_ticks.append("0.0")
+    left_y_ticks.append("1.0")
+    left_y_ticks.append("")
+    i = i + 1
+left_y_ticks.append("")
+
+# Create the tick labels for the right axis
+right_y_ticks = []
+right_y_ticks.append("")
+for year in warm:
+    right_y_ticks.append("")
+    right_y_ticks.append(year)
+    right_y_ticks.append("")
+    right_y_ticks.append("")
+right_y_ticks.append("")
+
+# Use geocat.viz.util convenience function to set axes limits & tick values without calling several matplotlib functions
+gvutil.set_axes_limits_and_ticks(
+    ax0,
+    xlim=(-3, 3),
+    xticks=np.linspace(-3, 3, 7),
+    xticklabels=["Jan₋₁", "Jul₋₁", "Jan₀", "Jul₀", "Jan₊₁", "Jul₊₁", "Jan₊₂"],
+    yticks=np.linspace(0, 46, 46),
+    yticklabels=left_y_ticks)
+
+ax0.spines['right'].set_visible(False)
+gvutil.set_axes_limits_and_ticks(axRHS,
+                                 ylim=(0, 45),
+                                 yticks=np.linspace(0, 45, 46),
+                                 yticklabels=right_y_ticks)
+
+# Set tick parameters for all axes
+ax0.tick_params(axis="x", length=9, labelsize=12)
+ax0.tick_params(axis="y", length=5, labelsize=8)
+axRHS.tick_params(axis="y", length=6, labelsize=8)
+
+# Show plot
+plt.plot()
+
+###############################################################################
+# Although the code above does recreate the NCL plot, it is tedious to replicate
```

Comment:
@anissa111 based on your comment from maponly, do you think we should have this block as part of its own cell of text, or keep it in this format?

Reply:
@michaelavs This is the format I was talking about, yeah! This example correctly formats the comment block for a markdown cell when rendered:

![image](https://user-images.githubusercontent.com/38434768/121267911-2bb85c80-c87a-11eb-95e3-4636c8a5ae4a.png)